content/pages/privacy.nl.md

```diff
@@ -59,10 +60,12 @@ Vrijwilligers die informatie verwerken namens DWH tekenen een geheimhoudersverkl
 
 **Extern** worden gegevens aan de volgende partijen versterkt:
```

Comment:
Ik zou kijken om die externe partijen er gewoon uit te laten, ondanks dat ik ze ooit zelf erin heb gestopt. Ik zie dat veel andere partijen (incl. de overheid zelf op overheid.nl dat in de volgende stijl doen):

> KOOP deelt uw gegevens niet met derden en verstrekt deze uitsluitend indien dit nodig is voor de uitvoering van onze overeenkomst met u of uw organisatie, om te voldoen aan hetgeen u verzoekt of om te voldoen aan een wettelijke verplichting. Met externe partijen die uw gegevens verwerken teneinde onze dienstverlening te kunnen realiseren, hebben wij verwerkersovereenkomsten gesloten om te zorgen voor eenzelfde niveau van beveiliging en vertrouwelijkheid van uw gegevens. KOOP en haar opdrachtgevers blijven verantwoordelijk voor deze verwerkingen.

Wij hebben ook verwerkersovereenkomsten met de relevante partijen